sdk/java-sdk/src/main/scala/com/akkaserverless/javasdk/impl/AnySupport.scala

```diff
@@ -389,8 +389,13 @@ class AnySupport(
         ScalaPbAny(BytesPrimitive.fullName, primitiveToBytes(BytesPrimitive, byteString))
 
       case other =>
-        throw SerializationException(
-          s"Don't know how to serialize object of type ${other.getClass}. Try passing a protobuf, using a primitive type, or using a type annotated with @Jsonable.")
+        try {
+          ScalaPbAny.fromJavaProto(JsonSupport.encodeJson(value))
+        } catch {
+          case _: IllegalArgumentException =>
+            throw SerializationException(
+              s"Don't know how to serialize object of type ${value.getClass}. Try passing a protobuf, using a primitive type, or using a type annotated with @Jsonable.")
```

Comment:
`IllegalArgumentException` is thrown from `encodeJson` if it can't serialize it, but maybe we want to catch all exceptions here.